api/spec/requests/spree/api/checkouts_controller_spec.rb

```diff
@@ -59,12 +59,8 @@ module Spree
       end
 
       it "will return an error if the order cannot transition" do
-        skip "not sure if this test is valid"
-        order.bill_address = nil
-        order.save
-        order.update_column(:state, "address")
+        order.update(state: 'address', ship_address: nil)
```

Comment:
Can you use `update!` instead of `update`? That way, if the spec setup fails for some reason, the test will break loudly instead of failing its expectation.